mvc/migration/migratingfrommvc5.rst

```diff
@@ -1,257 +1,244 @@
-
-.. code-block:: javascript
-
-	{
-		"name": "NewMvc6Project",
-		"private": true,
-		"dependencies": {
-			"bootstrap": "3.0.0",
-			"jquery": "1.10.2",
-			"jquery-validation": "1.11.1",
-			"jquery-validation-unobtrusive": "3.2.2"
-		},
-		"exportsOverride": {
-			"bootstrap": {
-				"js": "dist/js/*.*",
-				"css": "dist/css/*.*",
-				"fonts": "dist/fonts/*.*"
-			},
-
-			"jquery": {
-				"": "jquery.{js,min.js,min.map}"
-			},
-			"jquery-validation": {
-				"": "jquery.validate.js"
-			},
-			"jquery-validation-unobtrusive": {
-				"": "jquery.validate.unobtrusive.{js,min.js}"
-			}
-		}
-	}
-
-Bower will automatically download the specified dependencies, but for now the files are not yet in the wwwroot folder, and so cannot be requested by the browser:
-
-.. image:: migratingfrommvc5/_static/project-structure-bower.png
-
-Next, we will configure Gulp to process these files and place them where we want them in the wwwroot folder. First, we need to make sure Gulp is installed locally for the project. This is accomplished using NPM, which is similar to Bower but requires a different configuration file, "package.json". Add a new NPM configuration file to the root of the project, called package.json. Add *gulp*, *rimraf*, and *gulp-concat* to the devDependencies property (you should get Intellisense as you type each package name). When you're finished, your file should look similar to this one:
-
-.. literalinclude:: migratingfrommvc5/samples/NewMvc6Project/src/NewMvc6Project/package.json
-	:language: javascript
-	:emphasize-lines: 6-8
-	:linenos: 
+The MVC 5 project uses `Bootstrap <http://getbootstrap.com/>`__ for its styling, and stores the Bootstrap files in the *Content* and *Scripts* folders. The template generated MVC 5 project references Bootstrap in the layout file (*Views/Shared/_Layout.cshtml*). You could copy the *bootstrap.js* and *bootstrap.css* files from the MVC 5 project to the *wwwroot* folder in the new project, but that approach doesn't use the improved mechanism for managing client-side dependencies in ASP.NET 5.
 
-Save your changes. You should see a new NPM folder in your project, under Dependencies, and it should now include gulp (3.8.11) as well as the related packages. In addition to Gulp itself, these two packages will allow us to clean up folders before we copy files to them, and to concatenate two or more files together, to achieve bundling.
+In the new project, we'll add support for Bootstrap (and other client-side libraries) using `Bower <http://bower.io/>`__ and `Grunt <http://gruntjs.com/>`__.
 
-Next, add a new Gulp Configuration file (Gulpfile.js) to the root of the project.
+- Add a `Bower <http://bower.io/>`__ configuration file named *bower.json* to the project root (Right click on the project, and then **Add > New Item > Bower Configuration File**).
```

Comment:
Don't put instructions in ( ). Instead:
Add a `Bower <http://bower.io/>`__ configuration file named *bower.json* to the project root by right-clicking on the project, and then **Add > New Item > Bower Configuration File**.